epoch_tracker/epoch_tracker.cc

```diff
@@ -764,10 +764,10 @@ bool EpochTracker::ComputeFeatures(void) {
   n_feature_frames_ = bandpassed_rms_.size();
   float mean = 0.0;
   GetSymmetryStats(residual_, &positive_rms_, &negative_rms_, &mean);
-  fprintf(stdout, "Residual symmetry: P:%f  N:%f  MEAN:%f\n",
```

Comment:
We don't really want to leave commented out printfs in the code. If you want to propose some solutions that involves using some option or _DEBUG flag I'll be happy to have a look.